docs/tutorials/column_filtering.ipynb

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
<p>This still says "12 of 15 columns", so it's not loading all of them? Are there default columns in play here? Can we talk about that some? I think you can do <code>columns="all"</code> to get no-really-all the columns. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>